- tuple( [1,2] ) doesn't work (printer error)
- lexer doesn't support imaginary numbers, unicode, new @ syntax
- no metaclasses
- no properties
- generator functions not handled yet
- exceptions must be recast as python exceptions as much as possible (what about tracebacks?)
- lexer should allow newlines inside lists, tuples, dicts
- support   def f((a,b), c): print b
            f((1,2),3) -> "2"
  by introducing `anonymous' variable name for first argument.
  http://mail.python.org/pipermail/python-dev/2003-October/038915.html

- support non-modules in modules registry sys.modules
http://groups.google.com/groups?hl=en&lr=&c2coff=1&threadm=mailman.981282736.25263.python-list%40python.org&rnum=3&prev=/groups%3Fas_q%3Dauthor:peters%2520sys%2520modules%26safe%3Dimages%26ie%3DUTF-8%26oe%3DUTF-8%26lr%3D%26num%3D100%26hl%3Den

- break only allowed lexically inside for/while
>>> def f():
...   break
...
#<user-defined-function f @ #x71d9ad1a>
>>> for i in range(3):
...   f()


class d(dict):
   def __setitem__(self, key, val):
     dict.__setitem__(self, key.lower(), val)
   def __getitem__(self, key):
     dict.__getitem__(self, key.lower())

x = d(a=3, b=5)
# ignores keyword args, but should use them in __init__


20060302
--------

>>> class C: pass
... 
#<class C @ #x7ddf2302>
>>> x = C()
#<C @ #x7e45594a>
>>> x.__dict__
Error: AttributeError: No such attribute: #<C> `__dict__' (class: #<py-type C>)
  [condition type: AttributeError]

------------------------------------------------------------------------------
20060312

- Name mangling of __xxx inside classes (both names and attributes)
  is not implemented yet.

- Ensure correct calling of __setattr__, __delattr__, __set__,
  __get__, __del__.


------------------------------------------------------------------------------
20060319

- String exceptions (raise "foo") not supported (deprecated in CPython)



failing b5:


Wrong: no exception UnicodeError for py-string('abcÿ',)
Wrong: '#<#:C @ #x730774b2>' != '<C object>'
